CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Patches] fix `HOST_OFFLINE` error when shutdown a host [#5053] https://github.com/vatesfr/xen-orchestra/issues/5053 (PR [#5407] (https://github.com/vatesfr/xen-orchestra/pull/5407)
```

Comment:
I don't think it's necessary since it's not directly visible by users (we don't log errors trigger by `listMissingPatches`).